test/com/google/javascript/jscomp/PolymerBehaviorExtractorTest.java

```diff
@@ -28,8 +27,6 @@
  */
 public class PolymerBehaviorExtractorTest extends CompilerTypeTestCase {
 
-  private static final Joiner LINE_JOINER = Joiner.on('\n');
```

Comment:
You mean the one in `BaseJSTypeTestCase`? It's already named `LINE_JOINER` and is protected: https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/rhino/testing/BaseJSTypeTestCase.java#L58
I thought Polymer*Test should just use that one instead?